interview-questions.md

```diff
@@ -8,50 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: 
 
 Researched answer:
 
```

Comment:
When answering these interview questions, please remember this is your practice for interviews. Let's move beyond brief definitions. Do the research to add more content to your answer. Paraphrase and use your own words. Next week, let's focus on complete sentences that address the definition of the topic and your experience using the tool.